scripts/build_shapes.py

```diff
@@ -113,11 +171,17 @@ def get_GADM_layer(country_list, layer_id, geo_crs, update=False, outlogging=Fal
         geodf_temp = gpd.read_file(file_gpkg, layer="ADM_ADM_" + str(layer_id)).to_crs(
             geo_crs
         )
+        geodf_temp = gpd.read_file(file_gpkg, layer="ADM_ADM_" + str(layer_id)).to_crs(
+            geo_crs
+        )
 
-        # convert country name representation of the main country (GID_0 column)
-        geodf_temp["GID_0"] = [
-            three_2_two_digits_country(twoD_c) for twoD_c in geodf_temp["GID_0"]
-        ]
+        geodf_temp = filter_gadm(
+            geodf=geodf_temp,
+            layer=layer_id,
+            cc=country_code,
+            output_nonstd_to_csv=True,
```

Comment:
The option output_nonstd_to_csv should be set to false in my opinion.
I'd avoid to keep debug files for the normal user